.github/workflows/master.yml

```diff
@@ -7,6 +7,26 @@ on:
     branches: [ master ]
 
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.6
+    - name: Install dependencies
+      run: |
+        export GITHUB_WORKFLOW=1
+        INSTALL_LARGE_PYTHON_DEPS=true INSTALL_SMALL_PYTHON_DEPS=true source ./travis/install-common-deps.sh
+        pip install -r ./travis/lint-requirements.txt
+    - name: Run tests
+      run: |
+        export GITHUB_WORKFLOW=1
```

Comment:
Why overwrite `GITHUB_WORKFLOW` with `1`?

https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables

Reply:
oh, I didn't realize there was already such an environment variable set! Haha.